contracts/PowerIndexRouter.sol

```diff
@@ -766,6 +794,7 @@ contract PowerIndexRouter is PowerIndexRouterInterface, PowerIndexNaiveRouter {
   {
     require(_reserveRatioPct <= HUNDRED_PCT, "RR_GREATER_THAN_100_PCT");
     expectedStakeAmount = getExpectedStakeAmount(_reserveRatioPct, _leftOnPiToken, _totalStakedBalance, _share);
+    expectedStakeAmount = expectedStakeAmount.add(_addToExpectedStakeAmount.mul(_share).div(1 ether));
```

Comment:
It seems that `_addToExpectedStakeAmount` value is processed incorrectly, as this line doesn't account for reserve %. 
In this case consider adding `_addToExpectedStakeAmount` value to either `_leftOnPiToken` of `_totalStakedBalance` at the line above.